tracer/src/Datadog.Trace/IAST/Evidence.cs

```diff
@@ -0,0 +1,22 @@
+// <copyright file="Evidence.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Iast
+{
+    internal class Evidence
```

Comment:
Consider making this type a `readonly struct` if that would help reduce heap allocations.

Reply:
Done!